### PR TITLE
feature(fonts): use standalone package for fonts

### PR DESCRIPTION
**Pull request by nappynapster**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser-Support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser-Support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1,32 +1,4 @@
...
-    line-height: 1;
-
-    /* Better Font Rendering =========== */
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-}
-
 @import './lib/styles/styles';
-@import './lib/styles/icons';
+@import 'src/lib/styles/function-groups';
```
Comment (pweyrich) [inline src/styles.scss:2]:
```suggestion
@import './lib/styles/function-groups';
```

File: src/styles.scss [hunk 2]
```diff
@@ -1,32 +1,4 @@
...
-    font-weight: normal;
-    font-variant: normal;
-    text-transform: none;
-    line-height: 1;
-
-    /* Better Font Rendering =========== */
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-}
-
```
Comment (pweyrich) [inline src/styles.scss:28]: das steckt dann in der styles.css aus dem terra-icons repo?

File: src/lib/styles/function-groups.scss [hunk 3]
```diff
@@ -1,5 +1,3 @@
-@import 'fonts/plentyicons';
```
[the remark this answers is not in the document]
Comment (nappynapster) [reply src/lib/styles/function-groups.scss:1]: nope das passiert jetzt in der angular.json

File: src/styles.scss (hunk 2 above)
Comment (nappynapster) [reply src/styles.scss:28]: yes

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,8 @@
 -   **terra-checkbox** removed getter and setter for `value` and `valueChange` emitter. Use `ngModel` and `ngModelChange` instead.
 -   **alert** removed `TerraAlertComponent`. Use `AlertService` instead.
 -   **input** `TerraInputComponent`'s `validate()` method no longer shows error alerts when the input is invalid.
+-   **font** icons moved to [terra-icon npm package](https://www.npmjs.com/package/@plentymarkets/terra-icons)
```
[the remark this answers is not in the document]
Comment (nappynapster) [reply CHANGELOG.md]: wird in der readme von terra-icons beschrieben

[the remark this answers is not in the document]
Comment (pweyrich) [reply CHANGELOG.md]: Dann gerne einen link dorthin angeben

File: src/lib/styles/fonts/plentysans.scss [hunk 5]
```diff
@@ -1,31 +0,0 @@
-/**
```
Comment (pweyrich) [inline src/lib/styles/fonts/plentysans.scss:1]: Wofür war diese Datei da?

Comment (nappynapster) [reply src/lib/styles/fonts/plentysans.scss:1]: die plentysans.scss wurde in der fonts.glob.scss importiert. Die wurde ja auch gelöscht

Comment (pweyrich) [reply src/lib/styles/fonts/plentysans.scss:1]: Ja, aber welchen Nutzen hatte diese Datei?

**Review by TimWieder — APPROVED** [review]

**Review by TimWieder — CHANGES_REQUESTED** [review]
funktioniert noch nicht

**Review by TimWieder — APPROVED** [review]

**Review by TimWieder — DISMISSED** [review]

**Review by TimWieder — DISMISSED** [review]

**Review by Neill83 — APPROVED** [review]
getestet

**Review by Neill83 — APPROVED** [review]

**Review by MFrank84 — APPROVED** [review]

**Review by TimWieder — APPROVED** [review]

**Pull request merged**